nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/proxy/HelmProxyRecipe.groovy

```diff
@@ -90,6 +94,7 @@ class HelmProxyRecipe
       builder.route(new Route.Builder().matcher(matcher)
           .handler(timingHandler)
           .handler(securityHandler)
+          .handler(routingRuleHandler)
```

Comment:
I think that this handler should be after `formatHighAvailabilitySupportHandler `

Reply:
That is a good one. Thanks. Fixed